source/tokens.rs

```diff
@@ -70,6 +78,70 @@ token!(
     "The `AS` token.\n\nRepresent the alias operator, e.g. `use Foo\\Bar as Baz`."
 );
 token!(
+    ASSIGN: "=";
+    "The `ASSIGN` token.\n\nRepresent a binding of a value to a variable, e.g. `$x = 42`."
+);
+token!(
+    ASSIGN_REFERENCE: "=&";
```

Comment:
Removed in https://github.com/php/php-langspec/commit/854b2d1d01e5c3d3feec16eebf94f5481611a875.